src/api/app/models/package.rb

```diff
@@ -1161,6 +1161,8 @@ def patchinfo
   end
 
   def delete_file(name, opt = {})
+    raise ScmsyncReadOnly if scmsync.present?
```

Comment:
I would still handover the scm url to the user here, so osc users have a chance to see where they can proceed.

Reply:
@adrianschroeter Those methods are not used for the API. I added it to `source_controller.rb` and also provide the SCM URL.